recipes/ios-cmake/all/conandata.yml

```diff
@@ -5,3 +5,9 @@ sources:
   "4.0.0":
     url: "https://github.com/leetal/ios-cmake/archive/4.0.0.zip"
     sha256: "2e0dcf00cd46ae808d85666ce4ce38ecd8875925588647295d41e97c874ad0b3"
+  "4.1.0":
+    url: "https://github.com/leetal/ios-cmake/archive/4.1.0.zip"
+    sha256: "13f32d364c2130469b1780c7b2d4914bab29f20230871202eb43c4f6665ccd29"
```

Comment:
```suggestion
```

Reply:
Ack, applied your changed and rebased it to add 4.1.1 only.

Reply:
You can click the commit suggestion button  if you like its easier for me